src/dom/map-setup.ts

```diff
@@ -1,41 +1,48 @@
+import {
+  Circle,
+  LatLngExpression,
+  LayerGroup,
+  Map,
+  Polyline,
+  TileLayer,
+} from "leaflet";
+import { ILapDataset } from "../models/go-kart-types";
 import { lapTimer, stopTimer } from "../utility/timer";
 
-let map = L.map("map");
-const circles = L.layerGroup();
-let circle = L.circle();
+let map = new Map("map");
+const circles: LayerGroup = new LayerGroup([]);
+const initialLatLng: LatLngExpression = [-29.697911, 30.525229];
```

Comment:
Agreed, or use SA and then zoom in once the API call resolves.

Reply:
Lat and Longs are only returned once the first lap was selected, therefore this would be a bad user experience. Ideally the API would return a centroid for a certain race (Like for Idube Raceway).